src/components/ChatEntry.js

```diff
@@ -1,22 +1,34 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({ id, sender, body, timeStamp, liked, onLike }) => {
+  const heartStatus = liked ? '❤️' : '🤍';
 
-const ChatEntry = (props) => {
   return (
     <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+      <h2 className="entry-name">{sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{body}</p>
+        <p className="entry-time">
```

Comment:
I would recommend using a div here instead of a paragraph tag since it isn't a good semantic representation of the time stamp content. Or maybe not wrapping the <TimeStamp> component in another tag at all.